packages/app/src/components/DonateComponent.tsx

```diff
@@ -51,26 +51,39 @@ function DonateComponent({ collective }: DonateComponentProps) {
     navigate(`/profile/${address}`);
   }
 
-  const [currency, setCurrency] = useState<string>('G$');
   const [frequency, setFrequency] = useState<Frequency>(Frequency.Monthly);
   const [duration, setDuration] = useState(12);
   const [decimalDonationAmount, setDecimalDonationAmount] = useState(0);
 
   const tokenList = useTokenList();
+  const gdEnvSymbol =
+    Object.keys(tokenList).find((key) => {
+      if (key.startsWith('G$')) {
+        return tokenList[key].address.toLowerCase() === collective.rewardToken.toLowerCase();
+      } else return false;
+    }) || 'G$';
```

Comment:
```suggestion
  const gdEnvSymbol = Object.keys(tokenList).reduce((acc, key) => {
  return key.startsWith('G$') && tokenList[key].address.toLowerCase() === collective.rewardToken.toLowerCase()
    ? key
    : acc;
}, 'G$');
```

Reduce for readability